core/domain/platform_parameter_list_test.py

```diff
@@ -37,8 +37,16 @@ class ExistingPlatformParameterValidityTests(test_utils.GenericTestBase):
                             'diagnostic_test',
                             'serial_chapter_launch_curriculum_admin_view',
                             'show_translation_size',
+                            'is_improvements_tab_enabled',
+                            'learner_groups_are_enabled',
                             'promo_bar_enabled',
-                            'promo_bar_message']
+                            'promo_bar_message',
+                            'always_ask_learners_for_answer_details',
+                            'high_bounce_rate_task_state_bounce_'
```

Comment:
What's this line? Seems incomplete.

Also, there are 8 lines added here but I thought there were only 6 things being moved in this PR. Is that expected?

Also can you maybe sort this alphabetically so it's easier to audit?

Reply:
Ah! I forgot to add the parenthesis, there are 6 params only just the ones that were long I split them into two lines.
For sorting, I have only arranged the ones I added at the end of the list for easy review because currently the complete list is not sorted and if I rearrange them then it will become a bit hard to review but let me know.
Thanks